apps/src/StudioApp.js

```diff
@@ -1796,7 +1797,7 @@ StudioApp.prototype.setStartBlocks_ = function (config, loadLastAttempt) {
     loadLastAttempt = false;
   }
   var startBlocks = config.level.startBlocks || '';
-  if (loadLastAttempt) {
+  if (loadLastAttempt && config.levelGameName !== 'Jigsaw') {
```

Comment:
It would be nice if, instead of a specific reference to Jigsaw, we could check some kind of game-type metadata about whether the game had a run button (or some equivalent "does this check the solution continuously" flag).